extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -950,7 +950,8 @@ public function isPageIdMainPage( $pageId ) {
 	 * @return string
 	 */
 	public function getHostName() {
-		return substr( $this->getGlobal( 'Server' ), 7 );
+		preg_match( "/^https?:\/\/([\w\-\.]+)/", $this->getGlobal( 'Server' ), $matches );
```

Comment:
I believe the regex would be easier if we use `preg_replace`.
Also I remember some talks about support for language path for https migration. Is it still valid?

Reply:
wgServer will not contain language path